### PR TITLE
Potted Plant Hiding

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Adds in the ability to hide behind potted plants (also works for items, of course).

This is feature I always loved about Goon's plants/shrubbery--not only were they decorative, but they had a gameplay function as well; you could hide behind them and partially obscure your sprite in an attempt to ambush someone or make a quick dodge from security.

hiding!

### CODE REVIEW

**Comment by Spacemanspark** [comment]
:+1:, for the most part.

**Comment by Iamgoofball** [comment]
#999 
WHAT WILL #1000 BE

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -196,6 +196,7 @@
 	icon = 'icons/obj/flora/plants.dmi'
 	icon_state = "plant-1"
 	anchored = 0
+	layer = 5
```
Comment (tigercat2000) [inline code/game/objects/structures/flora.dm]: >manually defining as layer 5

**Comment by Keilden** [comment]
Can oversized things like mechs "hide" behind them?

**Comment by Spacemanspark** [comment]
Does a mech look like it would fit behind that sprite?
The Tajaran barely does.

**Comment by fj45** [comment]
AHHAHA, at some point I had thought about this, great.

**Pull request merged**